dockerfiles/proj-ci-without-system-proj

```diff
@@ -18,4 +18,5 @@ RUN apt-get update \
   && DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends \
     clang \
     cmake \
+    sqlite3 \
```

Comment:
I'm curious why the package dependencies have changed in all the images. They look to be deps for building proj, but why did it change?

Reply:
I was seeing some build failures last night – let me try removing it again.

Reply:
Passing now. I hate gremlins.